Anytype/Sources/PresentationLayer/Common/SwiftUI/Search/LinkToObject/LinToObjectSearchView.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import SwiftUI
+
+struct LinToObjectSearchView: View {
```

Comment:
LinToObjectSearchView -> LinkToObjectSearchView

Reply:
Renamed in next pr